<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BioPIndia</class>
 <widget class="QMainWindow" name="BioPIndia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1148</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>320</y>
      <width>421</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="webcam_grid"/>
   </widget>
   <widget class="QLabel" name="temperature_sensor_1_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Temperature Sensor 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="temperature_sensor_2_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Temperature Sensor 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="humidity_sensor_1_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Humidity Sensor 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="humidity_sensor_2_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Humidity Sensor 2</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="temp_1_lcd">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="humidity_1_lcd">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="temp_2_lcd">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="humidity_2_lcd">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>561</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="sensor_plot_grid"/>
   </widget>
   <widget class="QLabel" name="Time_label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>580</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Time(s)</string>
    </property>
   </widget>
   <widget class="QLabel" name="mean_t">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Mean</string>
    </property>
   </widget>
   <widget class="QLabel" name="temperature">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Temperature</string>
    </property>
   </widget>
   <widget class="QLabel" name="mean_h">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>330</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Mean</string>
    </property>
   </widget>
   <widget class="QLabel" name="humidity">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>350</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Humidity</string>
    </property>
   </widget>
   <widget class="QLabel" name="smoke">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>460</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Smoke</string>
    </property>
   </widget>
   <widget class="QLabel" name="concentration">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Concentration</string>
    </property>
   </widget>
   <widget class="QLabel" name="desired_temp_label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Desired Temperature</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="desired_temp_spinBox">
    <property name="geometry">
     <rect>
      <x>427</x>
      <y>5</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="desired_temp_set_pushbox">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QLabel" name="stepper_speed_label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Stepper Speed</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="mean_temp_lcd">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="mean_hum_lcd">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>370</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="smoke_conc_lcd">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>500</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="power_off_button">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Power Off</string>
    </property>
   </widget>
   <widget class="QSlider" name="stepper_speed_slider">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>70</y>
      <width>21</width>
      <height>81</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1148</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionView_Loaded_STL_File"/>
    <addaction name="actionGenerate_GCode_for_STL_File"/>
    <addaction name="actionPrint_via_RepetierHost"/>
    <addaction name="actionExit"/>
    <addaction name="actionLoad_STL_File"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionPost_process_GCode_for_BioP"/>
   </widget>
   <widget class="QMenu" name="menuPort">
    <property name="title">
     <string>Serial Port</string>
    </property>
    <addaction name="actionCOM"/>
    <addaction name="actionCOM2"/>
    <addaction name="actionCOM3"/>
   </widget>
   <widget class="QMenu" name="menuWebcam_Port">
    <property name="title">
     <string>Video Device</string>
    </property>
    <addaction name="action1"/>
    <addaction name="action2"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuPort"/>
   <addaction name="menuWebcam_Port"/>
  </widget>
  <action name="actionView_Loaded_STL_File">
   <property name="text">
    <string>View Loaded STL File</string>
   </property>
  </action>
  <action name="actionGenerate_GCode_for_STL_File">
   <property name="text">
    <string>Generate GCode for STL File</string>
   </property>
  </action>
  <action name="actionPrint_via_RepetierHost">
   <property name="text">
    <string>Print via RepetierHost</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPost_process_GCode_for_BioP">
   <property name="text">
    <string>Post process GCode for BioP</string>
   </property>
  </action>
  <action name="actionCOM">
   <property name="text">
    <string>COM1</string>
   </property>
  </action>
  <action name="actionCOM2">
   <property name="text">
    <string>COM2</string>
   </property>
  </action>
  <action name="actionCOM3">
   <property name="text">
    <string>COM3</string>
   </property>
  </action>
  <action name="actionLoad_STL_File">
   <property name="text">
    <string>Load STL File</string>
   </property>
  </action>
  <action name="action1">
   <property name="text">
    <string>USB 1</string>
   </property>
  </action>
  <action name="action2">
   <property name="text">
    <string>USB 2</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>BioPIndia</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
